src/Symfony/Component/Yaml/Inline.php

```diff
@@ -627,7 +627,7 @@ public static function evaluateBinaryScalar($scalar)
 
     private static function isBinaryString($value)
     {
-        return preg_match('/[^\x09-\x0d\x20-\xff]/', $value);
+        return !preg_match('//u', $value);
```

Comment:
In similar cases, I usually use:
`return !preg_match('//u', $value) || preg_match('/[\x00-\x08\x0B\x0E-\x1A\x1C-\x1F\x7F]+/', $value);`